Cargo.toml

```diff
@@ -26,7 +26,7 @@ log = "0.4"
 futures_codec = "0.4"
 
 [dev-dependencies]
-chrono = "^0.4"
+chrono = "0.4"
```

Comment:
Why do you pin it to certain release? I thought that using ^0.4 notation is semantically better cause it will automatically use newer version when available.

Reply:
Caret requirements are equivalent to requirements without the caret symbol. From the cargo guide:
>The string "0.1.12" is a semver version requirement. Since this string does not have any operators in it, it is interpreted the same way as if we had specified "^0.1.12", which is called a caret requirement.

From my brief survey of other open source libraries, I saw that it seemed to be the established convention to prefer the non-caret syntax, which is why I personally specify things with the non-caret sytax. However, both are identical - I'm good to do the caret one if you feel strongly on that. 
Sources cited: [rocket](https://github.com/SergioBenitez/Rocket/blob/master/core/lib/Cargo.toml), [thiserror](https://github.com/dtolnay/thiserror/blob/master/Cargo.toml), [actix](https://github.com/actix/actix/blob/master/Cargo.toml), [tokio](https://github.com/tokio-rs/tokio/blob/master/tokio/Cargo.toml), [log](https://github.com/rust-lang/log/blob/master/Cargo.toml), [rand](https://github.com/rust-random/rand/blob/master/Cargo.toml), [syn](https://github.com/dtolnay/syn/blob/master/Cargo.toml), [lazy_static](https://github.com/rust-lang-nursery/lazy-static.rs/blob/master/Cargo.toml), [futures](https://github.com/rust-lang/futures-rs/blob/master/futures/Cargo.toml), etc (I didn't find any examples with the caret symbol that I looked at)

Reply:
> Caret requirements are equivalent to requirements without the caret symbol.

It seems that I missed that part of the book..